core/src/epicli/data/common/ansible/playbooks/kubernetes_master.yml

```diff
@@ -7,15 +7,15 @@
   become: true
   become_method: sudo
   pre_tasks:
+    - import_role:
+        name: kubernetes_master
+        tasks_from: noop  # just to include role vars
```

Comment:
Wouldn't it be better to use the include_vars module?

Reply:
Please take a look a latest commits :+1: